### PR TITLE
It ridgeline example

### PR DESCRIPTION
**Pull request by iliatimofeev**
Ridgeline plot (Joyplot) Example
--------------------------------
A [Ridgeline plot](https://serialmentor.com/blog/2017/9/15/goodbye-joyplots) chart is a chart that lets you visualize distribution of a numeric value for 
several groups.

Such a chart can be created in Altair by first transforming the data into a suitable representation.
visualization - 2019-05-20T225810 490

### CODE REVIEW

**Comment by iliatimofeev** [comment]
I should be a bit simpler but due to https://github.com/vega/vega-lite/issues/4939 this code has some side effects  

```python
import altair as alt
from vega_datasets import data

source = data.seattle_weather.url

step = 20
overlap = 1

ridgeline = alt.Chart(source, height=step).mark_line(
    interpolate="monotone", fillOpacity=0.8, stroke="lightgray", strokeWidth=0.5
).encode(
    alt.X(
        "temp_max:Q",
        bin=True,      
        title="Maximum Daily Temperature (C)",
    ),
    alt.Y(
        "count(temp_max):Q",
        scale=alt.Scale(range=[step, -step * overlap]),
        impute=alt.ImputeParams(value=0),
        axis=None
    ),
    alt.Fill(
        "mean(temp_max):Q",
        legend=None,
        scale=alt.Scale(domain=[40, 5], scheme="redyellowblue")
    ),
    alt.Row(
        "date:T",
        timeUnit="month",
        title=None,
        header=alt.Header(labelAngle=0, labelAlign="right", format="%B")
    )
).properties(
    bounds="flush", title="Seattle Weather"
).configure_facet(
    spacing=0
).configure_view(
    stroke=None
).configure_title(
    anchor="end"
)

ridgeline
```

File: altair/examples/ridgeline_plot.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# category: other charts
+import altair as alt
+from vega_datasets import data
+
+source = data.seattle_weather.url
+
+step = 20
+overlap = 1
+
+ridgeline = alt.Chart(source, height=step).transform_timeunit(
```
Comment (jakevdp) [inline altair/examples/ridgeline_plot.py]: Let's put ``height=step`` into the ``properties()`` call below.

**Comment by iliatimofeev** [comment]
Original hack with scale and flush bounds from Jeff's [PSR B1919+21](https://observablehq.com/@vega/psr-b1919-21-with-vega-lite) 
Idea of current chat is adapted from [Temperatures in Lincoln NE in 2016](https://cran.r-project.org/web/packages/ggridges/vignettes/introduction.html)
Should I mention it somewhere?

File: altair/examples/ridgeline_plot.py (hunk 1 above)
Comment (iliatimofeev) [reply altair/examples/ridgeline_plot.py]: done

**Comment by jakevdp** [comment]
Thanks!

**Pull request merged**